src/main.rs

```diff
@@ -1098,6 +1141,11 @@ fn main() {
                         .help( "Generate webasm through Emscripten" )
                         .overrides_with( "target-asmjs-emscripten" )
                 )
+                .arg(
+                    Arg::with_name( "target-webasm-unknown" )
+                        .long( "target-webasm-unknown" )
+                        .help( "Generate webasm directly (wasm32-unknown-unknown, without Emscripten involvement)" )
```

Comment:
For now I'd be a good idea to add something like `HIGHLY EXPERIMENTAL` to the help message.

Reply:
Aye.